spec/swiftpm_spec.vim

```diff
@@ -1,38 +1,52 @@
 source spec/support/helpers.vim
 
 describe "SwiftPM"
-
-  before
-    cd spec/fixtures/swiftpm/
-  end
-
```

Comment:
Any reason for moving this `cd` statement out of the `before` block?

Reply:
Explicitness more than anything. I do think that eventually for swift-test we might want multiple fixtures, but I suppose it would still be useful to cd into the root of the fixture folder in the `before` block.

tbh I wrote this like 3 weeks ago before going on vacation and just stumbled across it this morning, so I've mostly forgotten my reasonings. I don't have a super strong opinion on this and am totally happy to drop this if you feel strongly about it.

Reply:
I would prefer to keep it in the `before` block for now, as we're only using a single directory so far. Later we can use `context` to set up different `before` blocks with different folders as needed.

Reply:
👍 Done!

Reply:
Great, thanks!